db/migrate/20180122172441_devise_create_users.rb

```diff
@@ -4,6 +4,11 @@ class DeviseCreateUsers < ActiveRecord::Migration[5.1]
   def change
     create_table :users do |t|
       ## Database authenticatable
+      t.string :user_name, null: false, unique: true
```

Comment:
you must create a _new_ migration to add these fields. Please do not edit migration files once they have been pushed to github.